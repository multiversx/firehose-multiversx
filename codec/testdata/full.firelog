FIRE BLOCK_BEGIN 6
FIRE BLOCK_END 6 ef2d127de37b942baad06145e54b0c619a1f22327b2ebbcfbec78f5564afe39d 1647087903319636000 12111a096d657461636861696e2204686173682a110a06657264317171120710041a03313030