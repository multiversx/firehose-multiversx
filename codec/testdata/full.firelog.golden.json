[{
  "height": 6,
  "prevHash": "ef2d127de37b942baad06145e54b0c619a1f22327b2ebbcfbec78f5564afe39d",
  "timestamp": 1647087903319636000,
  "MultiversxBlock": {
    "blockData": {
      "headerType": "metachain",
      "headerHash": "aGFzaA=="
    },
    "alteredAccounts": {
      "erd1qq": {
        "address": "",
        "nonce": 4,
        "balance": "100"
      }
    },
    "numberOfShards": 0,
    "highestFinalBlockNonce": 0
  }
}
]